start.py

```diff
@@ -0,0 +1,92 @@
+import click
+import subprocess
+
+INTELOWL_TAG_VERSION = "v1.9.1"
+
+docker_analyzers = ["thug", "apk_analyzers", "box_js", "static_analyzers"]
+
+path_mapping = {
+    "default": "docker/default.yml",
+    "test": "docker/test.override.yml",
+    "ci": "docker/ci.override.yml",
+    "traefik": "docker/traefik.override.yml",
+    "multi_queue": "docker/multi-queue.override.yml",
+    "apk_analyzers": "integrations/apk_analyzers/apk.yml",
+    "box_js": "integrations/box-js/boxjs.yml",
+    "static_analyzers": "integrations/static_analyzers/static_analyzers.yml",
+    "thug": "integrations/thug/thug.yml",
+    "apk_analyzers.test": "integrations/apk_analyzers/apk.test.yml",
+    "box_js.test": "integrations/box-js/boxjs.test.yml",
+    "static_analyzers.test": "integrations/static_analyzers/static_analyzers.test.yml",
+    "thug.test": "integrations/thug/thug.test.yml",
+}
+path_mapping["all_analyzers"] = [path_mapping[key] for key in docker_analyzers]
+path_mapping["all_analyzers.test"] = [
+    path_mapping[f"{key}.test"] for key in docker_analyzers
+]
+
+
+@click.command(
+    context_settings=dict(
+        ignore_unknown_options=True,
+        allow_extra_args=True,
+    )
+)
+@click.argument("mode", required=True, type=click.Choice(["prod", "test", "ci"]))
+@click.argument(
+    "docker_command", required=True, type=click.Choice(["build", "up", "down"])
+)
+@click.option("--traefik", default=False, is_flag=True)
```

Comment:
really think we should provide some help with each option. For example, `help="Uses the `traefik.override.yml` compose file"` and similarly for other options.